### PR TITLE
MRG: add pytables to travis-ci, Py35 full test run

### PR DESCRIPTION
**Pull request by matthew-brett**
Add a Python 3.5 run with full dependendencies.  Add (py)tables to full
dependencies.

Replaces #1101.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,6 +9,12 @@ language: python
 cache:
   directories:
     - $HOME/.cache/pip
+addons:
+  apt:
+    packages:
+      # For runs with pytables
+      - libhdf5-serial
```
Comment (arokem) [inline .travis.yml]: Maybe `libhdf5-serial-dev` ?

Comment (matthew-brett) [inline .travis.yml]: Whyso?

Comment (arokem) [inline .travis.yml]: Because it looks like apt-get is not finding this one.

Comment (arokem) [inline .travis.yml]: https://travis-ci.org/nipy/dipy/jobs/148146241#L158

**Comment by matthew-brett** [comment]
OK - changed that line - thanks.

**Comment by arokem** [comment]
OK - looks like it's doing the right thing: 

https://travis-ci.org/nipy/dipy/jobs/148199298#L1981

(we expect this to fail currently, see #1100)

OTOH, these look legit too: 

https://travis-ci.org/nipy/dipy/jobs/148199293
https://travis-ci.org/nipy/dipy/jobs/148199294

I am actually getting these on my laptop as well. It looks like some warning is raised that shouldn't be raised.

**Comment by arokem** [comment]
#1103 fixes these failures:

https://travis-ci.org/nipy/dipy/jobs/148199293
https://travis-ci.org/nipy/dipy/jobs/148199294

**Comment by arokem** [comment]
Now in https://github.com/matthew-brett/dipy/pull/4 instead.

**Comment by arokem** [comment]
Superseded by #1100

**Pull request merged**